### PR TITLE
[cisco_ise] Fix multiple pipeline processing issues

### PR DESCRIPTION
**Pull request by aleksmaus**
## Proposed commit message

Fix multiple pipeline processing issues, based on the users reported issues:

* Fix error: `field [_ingest._value] does not contain value_split [=]` 
* Fix error: `[grok_message_details: For input string: "05:00:02.595", Cannot invoke "java.lang.CharSequence.length()" because "this.wrapped" is null]`
* Fix error: `13.33.14.64, AdminSession=ProfilerSession' is not an IP string literal.`

**(!) The first fix requires changes in the elastic-package to handle validation on the field with `enabled:false`**

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).
- [x] I have verified that any added dashboard complies with Kibana's [Dashboard good practices](https://docs.elastic.dev/ux-guidelines/data-viz/dashboard-good-practices) 

## Related issues

- Requires https://github.com/elastic/elastic-package/pull/2192

### CODE REVIEW

**Comment by aleksmaus** [comment]
This PR will continue failing until the elastic-package is updated https://github.com/elastic/elastic-package/pull/2192

File: packages/cisco_ise/data_stream/log/elasticsearch/ingest_pipeline/pipeline_passed_authentications.yml [hunk 1]
```diff
@@ -140,21 +140,20 @@ processors:
...
-                value: '{{{_ingest.on_failure_message}}}'
-      ignore_missing: true
+      source: |-
+        def m = new HashMap();
+        for (val in ctx.cisco_ise.log.log_details['cisco-av-pair']) {
+          int pos = val.indexOf("=");
+          if (pos == -1) {
+            m[val] = null;
+          } else {
+            m[val.substring(0,pos)] = val.substring(pos+1);
```
Comment (mjwolf) [inline packages/cisco_ise/data_stream/log/elasticsearch/ingest_pipeline/pipeline_passed_authentications.yml:153]: `pos+1` could be outside the array bounds if `=` is at the last position. Which I guess is not likely to happen with valid data, but it's best to add a check for that

Comment (aleksmaus) [reply packages/cisco_ise/data_stream/log/elasticsearch/ingest_pipeline/pipeline_passed_authentications.yml:153]: This script is correct.
Here is a test example with this script and all possible data permutations.

```
POST /_ingest/pipeline/_simulate
{
  "pipeline": {
    "description": "_description",
    "processors": [
      {
        "script": {
          "source": """
          def m = new HashMap();
          for (val in ctx['cisco-av-pair']) {
            int pos = val.indexOf('=');
            if (pos == -1) {
              m[val] = null;
            } else {
              m[val.substring(0,pos)] = val.substring(pos+1);
            }
          }
          ctx.resolved = m
          """
        }
      }
    ]
  },
  "docs": [
    {
      "_index": "logs-cisco_asa.log-default",
      "_id": "id",
      "_source": {
        "cisco-av-pair": [
          "audit-session-id=CB7D4E0A00000FE896B0CB97",
          "64:63:2d:6f:70:61:71:75:65:3d:04:00:00:00:00:00:00:00:00:00:00:00",
          "novalue="
        ]
      }
    }
  ]
}
```

Result:

```
{
  "docs": [
    {
      "doc": {
        "_index": "logs-cisco_asa.log-default",
        "_version": "-3",
        "_id": "id",
        "_source": {
          "cisco-av-pair": [
            "audit-session-id=CB7D4E0A00000FE896B0CB97",
            "64:63:2d:6f:70:61:71:75:65:3d:04:00:00:00:00:00:00:00:00:00:00:00",
            "novalue="
          ],
          "resolved": {
            "audit-session-id": "CB7D4E0A00000FE896B0CB97",
            "64:63:2d:6f:70:61:71:75:65:3d:04:00:00:00:00:00:00:00:00:00:00:00": null,
            "novalue": ""
          }
        },
        "_ingest": {
          "timestamp": "2024-11-05T22:16:56.67698659Z"
        }
      }
    }
  ]
}
```

Comment (mjwolf) [reply packages/cisco_ise/data_stream/log/elasticsearch/ingest_pipeline/pipeline_passed_authentications.yml:153]: painless is weird

**Review by mjwolf — APPROVED** [review]

File: packages/cisco_ise/data_stream/log/elasticsearch/ingest_pipeline/pipeline_passed_authentications.yml (hunk 1 above)
Comment (aleksmaus) [reply packages/cisco_ise/data_stream/log/elasticsearch/ingest_pipeline/pipeline_passed_authentications.yml:153]: it's java `substring` implementation which is I think consistent with other languages as well

**Comment by aleksmaus** [comment]
/test

the two integrations tests not related to this PR changes keep failing

```
dependency failed to start: container elastic-package-stack-elastic-agent-1 exited (1)
--
  | Error: booting up the stack failed: running docker-compose failed: running command failed: running Docker Compose up command failed: exit status 1: Container elastic-package-stack-package-registry-1  Running
```

Screenshot 2024-11-11 at 3 03 21 PM

File: go.mod [hunk 2]
```diff
@@ -7,7 +7,7 @@ toolchain go1.22.5
 require (
 	github.com/blang/semver v3.5.1+incompatible
 	github.com/cli/go-gh/v2 v2.11.0
-	github.com/elastic/elastic-package v0.106.0
+	github.com/elastic/elastic-package v0.107.0
```
[the remark this answers is not in the document]
Comment (jsoriano) [reply go.mod:10]: Updating in https://github.com/elastic/integrations/pull/11709

**Pull request merged**